### DISCUSSION TITLE
Start menu not working in Win11 Build 25136 ?

### DISCUSSION THREAD

**Discussion by n000dles** [Open-Shell/Open-Shell-Menu#1044] [Q&A]
Hi, please confirm the Start menu for Open Shell does not work on Windows 11 latest build 25136? I installed v4.4.170 but the Start menu is still the default which comes Windows 11. 
I noticed it integrates with Windows Explorer but doesn't seem to work with replacing the Start menu

**Comment by MichaelIsGreat123** [comment]
Check this video: "(May 28, 2022) Change the Windows 11 start menu with Open-Shell - May 28, 2022 [by Sam Hayden] - YouTube" at https://www.youtube.com/watch?v=02RWTwZc9K0
In this video, the guy uses Open-Shell 4.4.160 and he has no problems at all installing successfully Open-Shell on Windows 11.

I currently cannot at all test this scenario because I am still on Windows 10. However, if among you, there are people who recently successfully installed Open-Shell on Windows 11, it would be great if you would not mind leaving a comment on this posting and indicate:
---The date you installed Open-Shell on Windows 11?
---Which version of Open-Shell that you have used?
---Have you had any update yet of Windows 11 and has any update of Windows 11 caused any problem with the installed Open-Shell on your Windows 11 computer?
Thanks in advance for any feedback that you will share.

Kind Regards
Michael
